src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "../api";
+import { Link } from "react-router-dom";
+
+const IndividualArticle = () => {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState({});
+
+  useEffect(() => {
+    getArticleById(article_id).then((articleFromApi) => {
+      setArticle(articleFromApi);
+    });
+  }, [article_id]);
+
+  const { title, body, topic, author, votes, created_at, comment_count } =
+    article;
+
+  return (
+    <div>
+      <h2>{title}</h2>
+      <h3>
+        Written by: {author} on {created_at}
+      </h3>
+      <p>{body}</p>
+      <p>
+        Topic: <Link to={`/articles?topic=${topic}`}>{topic}</Link>
```

Comment:
You linked to your article's endpoint and then added a query for the topic. This works but it is something that you would more likely do when interacting with an API, not the routing on the frontend of an application.

You have approached this in the right way in the Nav component but don't currently have the parametric endpoint set up 

articles/:topic

we can make use of useParams( ) here, which means we don't have to add API styles queries to our endpoints.